Presentation.Web/app/components/data-processing/tabs/data-processing-agreement-tab-main.ts

```diff
@@ -0,0 +1,64 @@
+﻿module Kitos.DataProcessing.Agreement.Edit.Main {
+    "use strict";
+
+    export class EditMainDataProcessingAgreementController {
+        static $inject: Array<string> = [
+            "dataProcessingAgreementService",
+            "user",
+            "$scope",
+            "notify",
+            "$state",
+            "hasWriteAccess",
+            "dataProcessingAgreement"
+        ];
+
+        constructor(
+            private dataProcessingAgreementService: Services.DataProcessing.IDataProcessingAgreementService,
+            private user: Services.IUser,
+            private $scope,
+            private notify,
+            private $state: angular.ui.IStateService,
+            private hasWriteAccess,
```

Comment:
den skal nok være public for at det giver mening for læseren af klassen for view tilgår den jo og ja det virker alligevel, men lige nu ligner det bare en ubrugt property. Du kan evt. smide et interface på din controller så det er tydligt at se hvilke dele der udstilles til view som viewModel og så slapper eslint også lidt af